modules/kuberay-cluster/outputs.tf

```diff
@@ -17,4 +17,8 @@ output "ray_cluster_uri" {
     length(data.kubernetes_service.head-svc.status[0].load_balancer) > 0 ? (
       length(data.kubernetes_service.head-svc.status[0].load_balancer[0].ingress) > 0 ?
   "${data.kubernetes_service.head-svc.status[0].load_balancer[0].ingress[0].ip}" : "") : "") : ""
+}
+
+output "ray_dashboard_uri" {
```

Comment:
this is not seprate then above one. - ray_cluster_uri .. merge this into above output

Reply:
This is separate only for the dashboard, it can not be used to submit job